xml/System.Text.Json.Serialization/JsonNumberHandling.xml

```diff
@@ -21,7 +21,7 @@
   </Attributes>
   <Docs>
     <summary>Determines how <see cref="T:System.Text.Json.JsonSerializer" /> handles numbers when serializing and deserializing.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>The fields of this enumeration are not defined by the JSON specification. Altering the default number handling can potentially produce JSON that cannot be parsed by other JSON implementations.</remarks>
```

Comment:
@deeprobin Is this remark meant to apply to `AllowNamedFloatingPointLiterals`? If so, it should be moved to line 46. Also, can you change it to say "This field is not defined by...".